apps/barista-design-system/src/components/ba-decision-graph/ba-decision-graph-node/ba-decision-graph-node.html

```diff
@@ -0,0 +1,66 @@
+<div class="ba-uxd-node-wrapper" *ngIf="_node">
+  <div #nodes *ngFor="let node of _decisionGraphSteps">
+    <div class="ba-uxd-node-container">
+      <div class="ba-uxd-node" *ngIf="!node.tasknode">
+        <div
+          class="ba-uxd-node-content"
+          [innerHTML]="getSanitizedNodeText(node.text)"
+        ></div>
+        <div *ngIf="node.path.length">
+          <button
+            class="ba-uxd-edge-button"
+            *ngFor="let edge of node.path.sort()"
+            [ngClass]="
+          {
+            'ba-uxd-yes-edge': edge.text.toLowerCase().includes('yes'),
```

Comment:
please do the checks in the component and keep the template as logic free and simple as possible